godtools/App/Features/Dashboard/Presentation/Tools/Subviews/ToolCard/ToolCardViewModel.swift

```diff
@@ -35,7 +35,7 @@ class ToolCardViewModel: ObservableObject {
                         
         name = tool.name
         category = tool.category
-        languageAvailability = tool.languageAvailability.availabilityString
+        languageAvailability = tool.languageAvailability?.availabilityString ?? ""
```

Comment:
Hey @rachaelblue as we move to dependency inversion and implementing business logic in the interface implementations, I do want to avoid adding fallback behavior like this in the ViewModel and instead keep logic like this strictly in the interface implementations.  
With languageAvailability being optional I'd suggest updating the ViewModel's Published property to be optional as well, this way it can directly consume data provided by the domain layer and we get the benefit that the View can also optionally render the language availability.

Reply:
Ok yeah good point, that makes sense!  Will look out for that in the future as well.